com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveClasspathsHandler.java

```diff
@@ -193,4 +196,13 @@ public void acceptSearchMatch(SearchMatch match) {
         }
         return resolved.toArray(new String[resolved.size()]);
     }
+
+    private static boolean isRuntime(final IClasspathEntry classpathEntry) {
+        for (IClasspathAttribute attribute : classpathEntry.getExtraAttributes()) {
+            if (IClasspathManager.SCOPE_ATTRIBUTE.equals(attribute.getName()) && "runtime".equals(attribute.getValue())) {
```

Comment:
I would not add a dependency to m2e.jdt just for that attribute. I suggest you copy the value directly here